subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -607,6 +607,10 @@ If file system watching does not work on your system, then continuous build won'
 In particular, continuous build does not work when using `--no-daemon`.
 See <<gradle_daemon.adoc#sec:daemon_watch_fs,Watching the file system>> for more information.
 
+When Gradle detects a change to the inputs, it will not trigger the build immediately.
+Instead, it will wait until no additional changes are detected for a certain period of time - the quiet period.
+You can configure the quiet period in milliseconds by the Gradle property `<<build_environment.adoc#sec:gradle_configuration_properties,org.gradle.continuous.quietperiod>>`.
```

Comment:
Is milliseconds the best way to do this? Another option would be to allow fractional seconds.

Reply:
👍